alerting/src/main/kotlin/org/opensearch/alerting/AlertService.kt

```diff
@@ -123,7 +130,9 @@ class AlertService(
     fun composeQueryLevelAlert(
```

Comment:
Should we combine these Alert compositions into a single logic for common pieces to reduce duplicate code? Not a PR blocker - we can add to the backlog in META

Reply:
the logic to save an alert is common across all monitors.

this is the logic to create a new alert which IMO should be in different methods and doesn't need a change as they all use different constructors to set their corresponding set of fields in alerts